src/magic/actmagic.cpp

```diff
@@ -3525,7 +3525,7 @@ void createParticleSap(Entity* parent)
 		}
 		entity->setUID(-3);
 
-		if ( sprite = 977 ) // boomerang
+		if ( sprite == 977 ) // boomerang
```

Comment:
Aha. I don't think we ever formally put this in our coding guidelines, but I think we've talked about putting rtypes on the left in equality checks...I don't like how they usually read, but hey, the compiler can catch typos like this for us.